MovieTime/MovieTime/Service/MTService.swift

```diff
@@ -0,0 +1,205 @@
+//
+//  MTService.swift
+//  MovieTime
+//
+//  Created by Enrique Melgarejo on 08/01/18.
+//  Copyright © 2018 Choynowski. All rights reserved.
+//
+
+import UIKit
+
+typealias MTHTTPHeaders = [String: String]
+typealias MTRequestResponse = ((MTResponse) -> Void)?
+typealias MTImageRequestResponse = ((UIImage?, MTNetworkError?) -> Void)?
```

Comment:
try to use parameter labels in tuples, they make your code much more readable. It looks like Objective-C when I read `completion(nil, nil)` for example. In Swift you could have `completion(image: nil, error: nil)` which feels much more comfortable!